### PR TITLE
Move RedisClient functions into DBConnector

### PR DESCRIPTION
**Pull request by qiluo-msft**
The member functions of class RedisClient fit well in the class DBConnector, and there is not benefit of standalone RedisClient class. So we are trying to deprecated the class RedisClient.

Steps:
- [ ] this PR
- [ ] rewrite code in sairedis, swss, etc.
- [ ] mark the class deprecated

### CODE REVIEW

File: common/redisclient.h [hunk 1]
```diff
@@ -22,77 +22,48 @@ class RedisClient
...
-        std::shared_ptr<std::string> get(const std::string &key);
+        void hmset(const std::string &key, InputIterator start, InputIterator stop) { return m_db->hmset(key, start, stop); }
 
-        std::shared_ptr<std::string> hget(const std::string &key, const std::string &field);
+        std::shared_ptr<std::string> get(const std::string &key) { return m_db->get(key); }
 
-        std::vector<std::shared_ptr<std::string>> mget(const std::vector<std::string> &keys);
+        std::shared_ptr<std::string> hget(const std::string &key, const std::string &field) { return m_db->hget(key, field); }
 
-        std::vector<std::shared_ptr<std::string>> hmget(const std::string &key, const std::vector<std::string> &fields);
```
Comment (prsunny) [inline common/redisclient.h:59]: so hmget is deprecated?

**Review by prsunny — APPROVED** [review]
lgtm

File: common/redisclient.h (hunk 1 above)
Comment (qiluo-msft) [reply common/redisclient.h:59]: It was never implemented. So I remove it from member function.

**Review by shi-su — APPROVED** [review]
Looks good to me.

**Pull request merged**